src/Couchbase.Lite.Tests.Shared/ReplicationTest.cs

```diff
@@ -1161,7 +1164,10 @@ public void TestConflictResolverMergeDoc()
                 _otherDB.Save(doc1aMutable);
             }
 
-            RunReplication(config, 0, 0);
+            RunReplication(config, 0, 0, documentReplicated: (sender, args) =>
+            {
+                conflictResolved.Should().Be(true, "Because the DocumentReplicationEvent will not be notified until the conflicted document is resolved.");
+            });
```

Comment:
This is not going to be needed.  All resolutions are required to finish before replicator stop.